ci-operator/jobs/openshift/release/openshift-release-release-4.10-periodics.yaml

```diff
@@ -1205,108 +1205,3 @@ periodics:
     - name: manifest-tool-local-pusher
       secret:
         secretName: manifest-tool-local-pusher
-- agent: kubernetes
-  cluster: build01
-  decorate: true
-  interval: 24h
-  labels:
-    job-release: "4.10"
-    pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: release-openshift-ocp-installer-e2e-aws-csi-4.10
```

Comment:
I see references to these jobs in `core-services/template-deprecation/_allowlist.yaml`: https://github.com/openshift/release/blob/523b9fde129be2d6da5bf0da5ce160b17f27973b/core-services/template-deprecation/_allowlist.yaml#L79

Reply:
I removed it from the _allowlist.yaml in a new commit. `make template-allowlist` then asked for removal of the template, so I removed it.